control/archaius/transfer.go

```diff
@@ -82,6 +83,9 @@ func SaveToCBCache(raw *model.HystrixConfig) {
 	newKeys := make(map[string]bool)
 	// if there is no config, none key will be updated
 	if raw != nil {
+		for _, v := range client.GetAllClient() {
```

Comment:
we have service level timeout config, must not overwrite all client configs with global config